src/famfs_cli.c

```diff
@@ -645,6 +645,11 @@ do_famfs_cli_cp(int argc, char *argv[])
 		famfs_cp_usage(argc, argv);
 		return -1;
 	}
+	if (interleave_param.nstrips > FAMFS_MAX_SIMPLE_EXTENTS) {
+		fprintf(stderr, "famfs cp error: Number of strips(%lld) should not be more than maximum allowed strips(%d) \n",
```

Comment:
Maybe shorten the new fprintf's so they layout nicely in no more than 99 columns

Reply:
Done.